src/uct/tcp/tcp_iface.c

```diff
@@ -37,6 +37,11 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    "call to non-blocking vector socket send",
    ucs_offsetof(uct_tcp_iface_config_t, max_iov), UCS_CONFIG_TYPE_ULONG},
 
+  {"MIN_AM_SHORTV", "2kb",
+   "Threshold for switching non-blocking send to IOV non-blocking\n"
```

Comment:
Threshold from switching from send() to sendv() for short active messages
(no need to mention non blocking)